### PR TITLE
Update documentation to reflect multiple SLA Configurations

### PR DESCRIPTION
**Pull request by paulOsinski**
The documented process for creating an SLA configuration is out of date, along with the screenshot.  

These documentation changes are intended to capture how SLA configurations currently work, since the ability to work with multiple SLA configurations has been added.

### CODE REVIEW

File: docs/content/en/usage/features.md [hunk 1]
```diff
@@ -414,14 +414,15 @@ Similar Findings
 DefectDojo allows you to maintain your security SLAs and automatically
 remind teams whenever a SLA is about to get breached, or is breached.
 
-Simply indicate in the `System Settings` for each severity, how many
-days teams have to remediate a finding.
+To use with SLAs, open the `System Settings` page and click the 'Enable Finding SLAs' checkbox.
```
Comment (manuel-sommer) [inline docs/content/en/usage/features.md:417]: To apply SLAs to findings,...

Comment (manuel-sommer) [inline docs/content/en/usage/features.md:417]: In order to apply SLAs to findings, open `System Settings` and check 'Enable Finding SLAs'.

**Review by manuel-sommer — CHANGES_REQUESTED** [review]

File: docs/content/en/usage/features.md (hunk 1 above)
Comment (paulOsinski) [reply docs/content/en/usage/features.md]: Thank you!

File: docs/content/en/usage/features.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
----
+f---
```
Comment (manuel-sommer) [inline docs/content/en/usage/features.md:1]: remove "f"

**Review by manuel-sommer — CHANGES_REQUESTED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

(assuming @manuel-sommer changes get done)

File: docs/content/en/usage/features.md (hunk 2 above)
Comment (cneill) [inline docs/content/en/usage/features.md:1]:
```suggestion
---
```

**Review by Maffooch — APPROVED** [review]

**Review by cneill — APPROVED** [review]

**Review by grendel513 — APPROVED** [review]

File: docs/content/en/usage/features.md (hunk 2 above)
Comment (paulOsinski) [reply docs/content/en/usage/features.md]: thanks Charles!

Comment (paulOsinski) [reply docs/content/en/usage/features.md]: Thank you! Sorry for missing that one

**Pull request merged**